cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,6 @@
+describe('headline copy validation', () => {
+  it('navigates to the W&C site and compares the header to a provided string', () => {
+    cy.visit('https://work.co')
+    cy.get(".Header1").should("have.text", "We design and ship digital products that transform companies.")
```

Comment:
use `data-test-id` attribute for more unique selector (Plus those attributes are added exactly for purpose of automation tests) 
Also, update README.md file with commands on how to install (dependencies) and run the tests

Reply:
@stefanworkco updated the README and used the `data-test-id` selector